docs/source/getting-started/intro-quickstart/index.rst

```diff
@@ -51,7 +51,12 @@ If you would like to see this quickstart in a purely notebook format, we have a
 Step 1 - Get MLflow
 -------------------
 
-MLflow is available on PyPI. If you don't already have it installed on your system, you can install it with:
+MLflow is available on PyPI.
+
+Installing Full release
```

Comment:
```suggestion
Installing Stable Release
```

nit: `Full` sounds like there is difference in package size like vs skinny.

Reply:
Agreed, this does sound a bit misleading :)